src/NuGet.Clients/NuGet.Tools/NuGetPackage.cs

```diff
@@ -632,12 +631,15 @@ private void ShowManageLibraryPackageDialog(object sender, EventArgs e)
 
                     MessageHelper.ShowWarningMessage(errorMessage, Resources.ErrorDialogBoxTitle);
                 }
-            });
+            }).FileAndForget(
+                          TelemetryUtility.CreateFileAndForgetEventName(
```

Comment:
```suggestion
                          FileAndForget or FireAndForget? 
```

Reply:
The specific method I am calling is an extension method. 
It's FileAndForget in there: https://github.com/microsoft/vs-threading/blob/master/doc/cookbook_vs.md#task-returning-fire-and-forget-methods

I will change my commit message to say file instead of fire and forget.